addons/modules/stringtable.xml

```diff
@@ -2341,5 +2341,8 @@
         <Key ID="STR_ZEN_Modules_NoCrewAvailable">
             <English>Vehicle must have other crew available</English>
         </Key>
+        <Key ID="STR_ZEN_Modules_Vegetation">
+            <English>Vegetation</English>
+        </Key>
```

Comment:
These should be available in the base game:

- `STR_a3_to_hideTerrainObjects6` - Buildings
- `STR_a3_to_hideTerrainObjects7` - Walls & Fences
- `STR_a3_to_hideTerrainObjects8` - Vegetation
- `STR_a3_to_hideTerrainObjects9` - Other

Reply:
I thought that was the case but I couldn't find those strings! I did my usual np++ search files through P drive. How did you find those?

Reply:
I looked at `\a3\modules_f\config.cpp`. These strings are used as tooltips for the `ObjectTypeFilterWithCounter` 3DEN attribute.

Reply:
261c9cf